homeassistant/components/yolink/binary_sensor.py

```diff
@@ -95,6 +100,18 @@ class YoLinkBinarySensorEntityDescription(BinarySensorEntityDescription):
             device.device_type == ATTR_DEVICE_WATER_METER_CONTROLLER
         ),
     ),
+    YoLinkBinarySensorEntityDescription(
+        key="water_running",
+        translation_key="water_running",
+        device_class=BinarySensorDeviceClass.RUNNING,
+        value=lambda state: state.get("waterFlowing") if state is not None else None,
+        should_update_entity=lambda value: value is not None,
```

Comment:
Why the should update entity?

Reply:
The integration will accept message forwarding from other channels(APP or API). When the valve control message is forwarded to the entity, the message does not contain the water flow status field.